blockchain/v2/io.go

```
@@ -104,11 +133,9 @@ func (sio *switchIO) trySwitchToConsensus(state state.State, blocksSynced int) {
 	}
 }
 
-func (sio *switchIO) broadcastStatusRequest(base int64, height int64) {
-	msgBytes := cdc.MustMarshalBinaryBare(&bcStatusRequestMessage{
-		Base:   base,
-		Height: height,
-	})
+func (sio *switchIO) broadcastStatusRequest(base, height int64) {
+	bm, _ := bc.MsgToProto(&bc.StatusRequestMessage{Base: base, Height: height})
```

why we ignore errors here?